source/developer/contribution-guide.md

```diff
@@ -11,7 +11,15 @@ Thank you for your interest in contributing to Mattermost. Here's the process:
 
 3. If you have questions, post them in [Mattermost forum](http://forum.mattermost.org/) or join the [Contributors](https://pre-release.mattermost.com/core/channels/tickets) channel and announce the ticket you'd like to work on so it can be assigned to you. 
 
-It's okay to submit PRs to fix obvious bugs or add small improvements, but anything that significantly changes behavior or user expectations [requires a Help Wanted issue opened by the core team](http://docs.mattermost.com/process/help-wanted.html) so that the change can be tested, documented and supported. 
+It is fine to submit a PR for a bug or an incremental improvement with less than 20 lines of code change without a Help Wanted issue if the core committers who review the PR agree that the change is small. Some examples include
```

Comment:
Ah hmm..that's a good point.

```
It is fine to submit a PR for a bug or an incremental improvement with less than 20 lines of code change without a Help Wanted issue if the change to existing behaviour is small. Some examples include
 - sample 1
 - sample 2
 - ...

Core committers who review the PR are entitled to reject the PR if it significantly changes behavior or user expectations, which [requires a Help Wanted issue opened by the core team](http://docs.mattermost.com/process/help-wanted.html) so that the change can be tested, documented and supported. 
```

// cc @prixone how do you feel about the above definition?

The use of `incremenetal` suits here in that it would typically be an improvement to existing behaviour rather than submitting a new improvement which might be small.